src/components/app/TabBar.js

```diff
@@ -2,17 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
 import React, { PureComponent, PropTypes } from 'react';
 import classNames from 'classnames';
 import Reorderable from '../shared/Reorderable';
 
 class TabBar extends PureComponent {
-  constructor(props) {
+  constructor(props: Props) {
     super(props);
-    this._mouseDownListener = this._mouseDownListener.bind(this);
+    (this: any)._mouseDownListener = this._mouseDownListener.bind(this);
```

Comment:
This is how we've been handling bound functions, and is currently our preferred way to do it.